### PR TITLE
aws_lambda_event_source_mapping: Support reading `starting_position` fields

### PR DESCRIPTION
**Pull request by gmpify**
### Description

After importing event source mapping resources, TF tries to replace the resource on next apply. Seems this is expected behaviour, according to [official aws_lambda_event_source_mapping documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lambda_event_source_mapping#import).

Checking for a more recent [GetEventSourceMapping API docs](https://docs.aws.amazon.com/lambda/latest/dg/API_GetEventSourceMapping.html) though, seems like `starting_position` and `starting_position_timestamp` are now exposed on that API. (Don't know when the change was done on AWS side, but docs [were updated on March 19th](https://github.com/awsdocs/aws-lambda-developer-guide/commit/9cddb205087dc9739a3e83c66258715c7f53a783#diff-2bacb74e2c7882e04268eceb74764cb3b019ae7be9d1af3c829347265c6c8061R235-R236))

This PR adds support for TF to read fields `starting_position` and `starting_position_timestamp`; which in turn should avoid a recreation after import.

### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates #1133

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccAWSLambdaEventSourceMapping_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSLambdaEventSourceMapping_ -timeout 180m
=== RUN   TestAccAWSLambdaEventSourceMapping_kinesis_basic
=== PAUSE TestAccAWSLambdaEventSourceMapping_kinesis_basic
=== RUN   TestAccAWSLambdaEventSourceMapping_kinesis_removeBatchSize
=== PAUSE TestAccAWSLambdaEventSourceMapping_kinesis_removeBatchSize
=== RUN   TestAccAWSLambdaEventSourceMapping_sqs_basic
=== PAUSE TestAccAWSLambdaEventSourceMapping_sqs_basic
=== RUN   TestAccAWSLambdaEventSourceMapping_sqs_withFunctionName
=== PAUSE TestAccAWSLambdaEventSourceMapping_sqs_withFunctionName
=== RUN   TestAccAWSLambdaEventSourceMapping_SQSBatchWindow
=== PAUSE TestAccAWSLambdaEventSourceMapping_SQSBatchWindow
=== RUN   TestAccAWSLambdaEventSourceMapping_disappears
=== PAUSE TestAccAWSLambdaEventSourceMapping_disappears
=== RUN   TestAccAWSLambdaEventSourceMapping_changesInEnabledAreDetected
=== PAUSE TestAccAWSLambdaEventSourceMapping_changesInEnabledAreDetected
=== RUN   TestAccAWSLambdaEventSourceMapping_StartingPositionTimestamp
=== PAUSE TestAccAWSLambdaEventSourceMapping_StartingPositionTimestamp
=== RUN   TestAccAWSLambdaEventSourceMapping_KinesisBatchWindow
=== PAUSE TestAccAWSLambdaEventSourceMapping_KinesisBatchWindow
=== RUN   TestAccAWSLambdaEventSourceMapping_ParallelizationFactor
=== PAUSE TestAccAWSLambdaEventSourceMapping_ParallelizationFactor
=== RUN   TestAccAWSLambdaEventSourceMapping_MaximumRetryAttempts
=== PAUSE TestAccAWSLambdaEventSourceMapping_MaximumRetryAttempts
=== RUN   TestAccAWSLambdaEventSourceMapping_MaximumRetryAttemptsZero
=== PAUSE TestAccAWSLambdaEventSourceMapping_MaximumRetryAttemptsZero
=== RUN   TestAccAWSLambdaEventSourceMapping_MaximumRetryAttemptsNegativeOne
=== PAUSE TestAccAWSLambdaEventSourceMapping_MaximumRetryAttemptsNegativeOne
=== RUN   TestAccAWSLambdaEventSourceMapping_MaximumRecordAgeInSeconds
=== PAUSE TestAccAWSLambdaEventSourceMapping_MaximumRecordAgeInSeconds
=== RUN   TestAccAWSLambdaEventSourceMapping_MaximumRecordAgeInSecondsNegativeOne
=== PAUSE TestAccAWSLambdaEventSourceMapping_MaximumRecordAgeInSecondsNegativeOne
=== RUN   TestAccAWSLambdaEventSourceMapping_BisectBatch
=== PAUSE TestAccAWSLambdaEventSourceMapping_BisectBatch
=== RUN   TestAccAWSLambdaEventSourceMapping_KinesisDestinationConfig
=== PAUSE TestAccAWSLambdaEventSourceMapping_KinesisDestinationConfig
=== RUN   TestAccAWSLambdaEventSourceMapping_MSK
=== PAUSE TestAccAWSLambdaEventSourceMapping_MSK
=== CONT  TestAccAWSLambdaEventSourceMapping_kinesis_basic
=== CONT  TestAccAWSLambdaEventSourceMapping_MaximumRetryAttempts
=== CONT  TestAccAWSLambdaEventSourceMapping_MaximumRecordAgeInSecondsNegativeOne
=== CONT  TestAccAWSLambdaEventSourceMapping_MSK
=== CONT  TestAccAWSLambdaEventSourceMapping_SQSBatchWindow
=== CONT  TestAccAWSLambdaEventSourceMapping_sqs_basic
=== CONT  TestAccAWSLambdaEventSourceMapping_kinesis_removeBatchSize
=== CONT  TestAccAWSLambdaEventSourceMapping_BisectBatch
=== CONT  TestAccAWSLambdaEventSourceMapping_MaximumRecordAgeInSeconds
=== CONT  TestAccAWSLambdaEventSourceMapping_MaximumRetryAttemptsNegativeOne
=== CONT  TestAccAWSLambdaEventSourceMapping_MaximumRetryAttemptsZero
=== CONT  TestAccAWSLambdaEventSourceMapping_disappears
=== CONT  TestAccAWSLambdaEventSourceMapping_sqs_withFunctionName
=== CONT  TestAccAWSLambdaEventSourceMapping_ParallelizationFactor
=== CONT  TestAccAWSLambdaEventSourceMapping_KinesisBatchWindow
=== CONT  TestAccAWSLambdaEventSourceMapping_StartingPositionTimestamp
=== CONT  TestAccAWSLambdaEventSourceMapping_changesInEnabledAreDetected
=== CONT  TestAccAWSLambdaEventSourceMapping_KinesisDestinationConfig
--- PASS: TestAccAWSLambdaEventSourceMapping_sqs_withFunctionName (115.35s)
--- PASS: TestAccAWSLambdaEventSourceMapping_disappears (119.64s)
--- PASS: TestAccAWSLambdaEventSourceMapping_MaximumRecordAgeInSeconds (140.75s)
--- PASS: TestAccAWSLambdaEventSourceMapping_MaximumRetryAttempts (140.82s)
--- PASS: TestAccAWSLambdaEventSourceMapping_kinesis_basic (151.85s)
--- PASS: TestAccAWSLambdaEventSourceMapping_ParallelizationFactor (165.77s)
--- PASS: TestAccAWSLambdaEventSourceMapping_MaximumRecordAgeInSecondsNegativeOne (183.78s)
--- PASS: TestAccAWSLambdaEventSourceMapping_SQSBatchWindow (186.92s)
--- PASS: TestAccAWSLambdaEventSourceMapping_StartingPositionTimestamp (188.55s)
--- PASS: TestAccAWSLambdaEventSourceMapping_BisectBatch (209.52s)
--- PASS: TestAccAWSLambdaEventSourceMapping_KinesisDestinationConfig (209.56s)
--- PASS: TestAccAWSLambdaEventSourceMapping_changesInEnabledAreDetected (216.80s)
--- PASS: TestAccAWSLambdaEventSourceMapping_MaximumRetryAttemptsZero (228.50s)
--- PASS: TestAccAWSLambdaEventSourceMapping_KinesisBatchWindow (239.18s)
--- PASS: TestAccAWSLambdaEventSourceMapping_kinesis_removeBatchSize (254.41s)
--- PASS: TestAccAWSLambdaEventSourceMapping_MaximumRetryAttemptsNegativeOne (280.49s)
--- PASS: TestAccAWSLambdaEventSourceMapping_sqs_basic (294.44s)
--- PASS: TestAccAWSLambdaEventSourceMapping_MSK (1981.55s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	1983.756s
```

### CODE REVIEW

File: aws/resource_aws_lambda_event_source_mapping.go [hunk 1]
```diff
@@ -323,6 +323,14 @@ func resourceAwsLambdaEventSourceMappingRead(d *schema.ResourceData, meta interf
 		return fmt.Errorf("error setting topics: %w", err)
 	}
 
+	if eventSourceMappingConfiguration.StartingPosition != nil {
+		d.Set("starting_position", eventSourceMappingConfiguration.StartingPosition)
+	}
```
Comment (gmpify) [inline aws/resource_aws_lambda_event_source_mapping.go:328]: This makes change backwards compatible; so TF won't try to recreate event source mappings that currently have `nil` value on state. Maybe it's better to remove the `if` and allow value to get empty string? That would then cause a recreation of the resource.

Comment (ewbankkit) [reply aws/resource_aws_lambda_event_source_mapping.go]: We should be safe as currently differences between `null` and the default value `""` are suppressed.
I will change this during merge.

**Comment by ewbankkit** [comment]
@gmpify Thanks for the contribution 👏.
I went ahead and made the change mentioned in the conversation above and also simplified the acceptance test configurations.

**Review by ewbankkit — APPROVED** [review]
LGTM :rocket:.

##### Commercial

```console
% make testacc TEST=./aws TESTARGS='-run=TestAccAWSLambdaEventSourceMapping_'                                                      
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSLambdaEventSourceMapping_ -timeout 180m
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_basic
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_basic
=== RUN   TestAccAWSLambdaEventSourceMapping_SQS_basic
=== PAUSE TestAccAWSLambdaEventSourceMapping_SQS_basic
=== RUN   TestAccAWSLambdaEventSourceMapping_DynamoDB_basic
=== PAUSE TestAccAWSLambdaEventSourceMapping_DynamoDB_basic
=== RUN   TestAccAWSLambdaEventSourceMapping_SQS_BatchWindow
=== PAUSE TestAccAWSLambdaEventSourceMapping_SQS_BatchWindow
=== RUN   TestAccAWSLambdaEventSourceMapping_disappears
=== PAUSE TestAccAWSLambdaEventSourceMapping_disappears
=== RUN   TestAccAWSLambdaEventSourceMapping_SQS_changesInEnabledAreDetected
=== PAUSE TestAccAWSLambdaEventSourceMapping_SQS_changesInEnabledAreDetected
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_StartingPositionTimestamp
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_StartingPositionTimestamp
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_BatchWindow
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_BatchWindow
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_ParallelizationFactor
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_ParallelizationFactor
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttempts
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttempts
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttemptsZero
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttemptsZero
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttemptsNegativeOne
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttemptsNegativeOne
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRecordAgeInSeconds
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRecordAgeInSeconds
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRecordAgeInSecondsNegativeOne
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRecordAgeInSecondsNegativeOne
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_BisectBatch
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_BisectBatch
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_DestinationConfig
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_DestinationConfig
=== RUN   TestAccAWSLambdaEventSourceMapping_MSK
=== PAUSE TestAccAWSLambdaEventSourceMapping_MSK
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_basic
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttempts
=== CONT  TestAccAWSLambdaEventSourceMapping_MSK
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_DestinationConfig
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_BisectBatch
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_BatchWindow
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttemptsZero
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_ParallelizationFactor
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRecordAgeInSecondsNegativeOne
=== CONT  TestAccAWSLambdaEventSourceMapping_SQS_BatchWindow
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRecordAgeInSeconds
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttemptsNegativeOne
=== CONT  TestAccAWSLambdaEventSourceMapping_SQS_changesInEnabledAreDetected
=== CONT  TestAccAWSLambdaEventSourceMapping_DynamoDB_basic
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_StartingPositionTimestamp
=== CONT  TestAccAWSLambdaEventSourceMapping_SQS_basic
=== CONT  TestAccAWSLambdaEventSourceMapping_disappears
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_BisectBatch (79.31s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttempts (79.34s)
--- PASS: TestAccAWSLambdaEventSourceMapping_DynamoDB_basic (80.84s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRecordAgeInSeconds (83.32s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_BatchWindow (90.16s)
--- PASS: TestAccAWSLambdaEventSourceMapping_SQS_BatchWindow (94.76s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_StartingPositionTimestamp (129.37s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_ParallelizationFactor (129.45s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRecordAgeInSecondsNegativeOne (135.72s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_DestinationConfig (148.77s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttemptsNegativeOne (174.03s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttemptsZero (181.81s)
--- PASS: TestAccAWSLambdaEventSourceMapping_SQS_basic (183.65s)
--- PASS: TestAccAWSLambdaEventSourceMapping_SQS_changesInEnabledAreDetected (185.02s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_basic (190.09s)
=== CONT  TestAccAWSLambdaEventSourceMapping_disappears
--- PASS: TestAccAWSLambdaEventSourceMapping_disappears (69.40s)
--- PASS: TestAccAWSLambdaEventSourceMapping_MSK (1916.28s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	1919.431s
```

##### GovCloud

```console
% make testacc TEST=./aws TESTARGS='-run=TestAccAWSLambdaEventSourceMapping_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSLambdaEventSourceMapping_ -timeout 180m
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_basic
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_basic
=== RUN   TestAccAWSLambdaEventSourceMapping_SQS_basic
=== PAUSE TestAccAWSLambdaEventSourceMapping_SQS_basic
=== RUN   TestAccAWSLambdaEventSourceMapping_DynamoDB_basic
=== PAUSE TestAccAWSLambdaEventSourceMapping_DynamoDB_basic
=== RUN   TestAccAWSLambdaEventSourceMapping_SQS_BatchWindow
=== PAUSE TestAccAWSLambdaEventSourceMapping_SQS_BatchWindow
=== RUN   TestAccAWSLambdaEventSourceMapping_disappears
=== PAUSE TestAccAWSLambdaEventSourceMapping_disappears
=== RUN   TestAccAWSLambdaEventSourceMapping_SQS_changesInEnabledAreDetected
=== PAUSE TestAccAWSLambdaEventSourceMapping_SQS_changesInEnabledAreDetected
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_StartingPositionTimestamp
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_StartingPositionTimestamp
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_BatchWindow
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_BatchWindow
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_ParallelizationFactor
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_ParallelizationFactor
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttempts
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttempts
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttemptsZero
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttemptsZero
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttemptsNegativeOne
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttemptsNegativeOne
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRecordAgeInSeconds
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRecordAgeInSeconds
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRecordAgeInSecondsNegativeOne
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRecordAgeInSecondsNegativeOne
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_BisectBatch
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_BisectBatch
=== RUN   TestAccAWSLambdaEventSourceMapping_Kinesis_DestinationConfig
=== PAUSE TestAccAWSLambdaEventSourceMapping_Kinesis_DestinationConfig
=== RUN   TestAccAWSLambdaEventSourceMapping_MSK
=== PAUSE TestAccAWSLambdaEventSourceMapping_MSK
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_basic
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttempts
=== CONT  TestAccAWSLambdaEventSourceMapping_SQS_changesInEnabledAreDetected
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_BatchWindow
=== CONT  TestAccAWSLambdaEventSourceMapping_disappears
=== CONT  TestAccAWSLambdaEventSourceMapping_DynamoDB_basic
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_BisectBatch
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_ParallelizationFactor
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttemptsNegativeOne
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRecordAgeInSeconds
=== CONT  TestAccAWSLambdaEventSourceMapping_SQS_BatchWindow
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_StartingPositionTimestamp
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRecordAgeInSecondsNegativeOne
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_DestinationConfig
=== CONT  TestAccAWSLambdaEventSourceMapping_SQS_basic
=== CONT  TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttemptsZero
=== CONT  TestAccAWSLambdaEventSourceMapping_MSK
--- PASS: TestAccAWSLambdaEventSourceMapping_SQS_BatchWindow (66.56s)
--- PASS: TestAccAWSLambdaEventSourceMapping_DynamoDB_basic (75.94s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_DestinationConfig (103.19s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttemptsNegativeOne (107.05s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRecordAgeInSecondsNegativeOne (107.53s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttemptsZero (108.26s)
--- PASS: TestAccAWSLambdaEventSourceMapping_SQS_changesInEnabledAreDetected (109.75s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_BatchWindow (120.81s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRetryAttempts (126.59s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_ParallelizationFactor (139.70s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_StartingPositionTimestamp (142.33s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_BisectBatch (153.10s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_basic (159.54s)
--- PASS: TestAccAWSLambdaEventSourceMapping_Kinesis_MaximumRecordAgeInSeconds (165.74s)
--- PASS: TestAccAWSLambdaEventSourceMapping_SQS_basic (195.56s)
=== CONT  TestAccAWSLambdaEventSourceMapping_disappears
--- PASS: TestAccAWSLambdaEventSourceMapping_disappears (69.40s)
=== CONT  TestAccAWSLambdaEventSourceMapping_MSK
    provider_test.go:1072: skipping test for aws-us-gov/us-gov-west-1: Error running apply: exit status 1
        2021/05/07 17:16:06 [DEBUG] Using modified User-Agent: Terraform/0.12.30 HashiCorp-terraform-exec/0.13.3
        
        Error: error creating Lambda Event Source Mapping (arn:aws-us-gov:kafka:us-gov-west-1:357342307427:cluster/tf-acc-test-1472047573752645107/47b7322b-9075-4009-8b91-c4975ee9edcb-3): InvalidParameterValueException: Unsupported source arn : arn:aws-us-gov:kafka:us-gov-west-1:357342307427:cluster/tf-acc-test-1472047573752645107/47b7322b-9075-4009-8b91-c4975ee9edcb-3
        {
          RespMetadata: {
            StatusCode: 400,
            RequestID: "713c2b4b-dc9e-47df-b3f3-9009b92bcdfd"
          },
          Message_: "Unsupported source arn : arn:aws-us-gov:kafka:us-gov-west-1:357342307427:cluster/tf-acc-test-1472047573752645107/47b7322b-9075-4009-8b91-c4975ee9edcb-3",
          Type: "User"
        }
        
          on terraform_plugin_test.tf line 116, in resource "aws_lambda_event_source_mapping" "test":
         116: resource "aws_lambda_event_source_mapping" "test" {
        
        
--- SKIP: TestAccAWSLambdaEventSourceMapping_MSK (1435.60s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	1438.674s
```

File: aws/resource_aws_lambda_event_source_mapping.go (hunk 1 above)
Comment (gmpify) [reply aws/resource_aws_lambda_event_source_mapping.go]: Perfect, thanks much!

**Pull request merged**